content/blog/getting-started-with-spark-on-mapr-sandbox.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Getting Started with Spark on MapR Sandbox
+date: 2021-10-14T06:01:00.000Z
+author: MapR Tutorials
+authorimage: /img/Avatar1.svg
+---
```

Comment:
Please add tag "hpe-ezmeral-data-fabric" for this blog post.

Reply:
done